rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/FindFilesNotClassifiedOrHinted.java

```diff
@@ -66,7 +67,7 @@ public Object exec(@SuppressWarnings("rawtypes") List arguments) throws Template
         for (Vertex v : result)
         {
             FileModel f = context.getFramed().frame(v, FileModel.class);
-            if (f instanceof JavaSourceFileModel || f instanceof XmlFileModel)
+            if (f instanceof JavaSourceFileModel || f instanceof XmlFileModel || f instanceof JavaClassFileModel)
```

Comment:
Yes, they can if they are unparseable. I'm not sure if this works, but we should display those without a link.